tests/js/client/load-balancing/load-balancing-key-generator-noauth-cluster.js

```diff
@@ -90,18 +90,60 @@ function KeyGeneratorSuite () {
     return res;
   }
 
+  function generateCollectionDocs(name) {
+    // check that the generated keys are sequential when we send the requests
+    // via multiple coordinators.
+    db._create(name, {keyOptions: {type: "autoincrement"}});
+    assertNotEqual("", db[name].properties().distributeShardsLike);
+    let lastKey = null;
+    let url = "/_db/" + cn + "/_api/document/" + name;
+    // send documents to both coordinators
+    let i = 0;
+    let startingKey = null;
+    if (Number(name[name.length - 1]) === 1) {
+      startingKey = "abc123";
+    } else if (Number(name[name.length - 1]) === 2) {
+      startingKey = "150000";
+    }
+    for (i; i < 100000; ++i) {
+      let doc = {};
+      if (startingKey === null || i > 0) {
+        doc = {"x": 0, "c": 1};
+      } else {
+        doc = {"_key": `${startingKey}`, "x": 0, "c": 1};
+      }
+      let result = sendRequest('POST', url, /*payload*/ {"x": 0, "c": 1}, {}, i % 2 === 0);
```

Comment:
`x` and `c` are not relevant for this test.
```suggestion
      let result = sendRequest('POST', url, /*payload*/ {}, {}, i % 2 === 0);
```